frontend/src/api/problem.ts

```diff
@@ -1,4 +1,4 @@
-type Problem = {
+export type Problem = {
```

Comment:
Comment on the whole `problem.ts` file: I think the merge conflicts was likely because I changed this file name to `Problem.ts` instead of `problem.ts`. That matches with the file naming pattern -- can you change this file name to `Problem.ts`?